src/main/java/core/basesyntax/service/FruitsService.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Operations;
+import java.util.List;
+
+public interface FruitsService {
+    void createNewFruits(String fruitName, int quantity);
+
+    int getOperatedCount(int oldFruitsQuantity, int newFruitsQuantity, Operations operation);
+
+    void createFruitsFromList(List<String> fruitsDocumentation);
```

Comment:
The method 'createFruitsFromList' is ambiguous. It should be named more clearly to indicate what it does with the list of fruit documentation. If it's parsing and adding them to storage, consider a name like 'processFruitTransactions' or 'updateStorageWithFruitList'.